src/pages/transactions/index.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @link https://github.com/invoiceninja/invoiceninja source repository
+ *
+ * @copyright Copyright (c) 2022. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+export { default as Transactions } from './index/Transactions';
+export { default as CreateTransaction } from './create/Create';
+export { default as EditTransaction } from './edit/Edit';
+export { default as Transaction } from './show/Transaction';
```

Comment:
Please use named exports, so you don't need files like these.

Reply:
> Please use named exports, so you don't need files like these.

done